src/language-css/printer-postcss.js

```diff
@@ -123,24 +120,25 @@ function genericPrint(path, options, print) {
       }
 
       return [
-        node.raws.before.replaceAll(/[\s;]/gu, ""),
+        node.raws.before?.replaceAll(/[\s;]/gu, "") ?? "",
```

Comment:
Because `raws` are not yet implemented in sass-parser, some inline comments are currently being inadvertently removed.